plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraRecordCursor.java

```diff
@@ -68,7 +66,7 @@ public boolean getBoolean(int i)
     @Override
     public long getCompletedBytes()
     {
-        return count;
+        return 0;
```

Comment:
The `io.trino.spi.connector.RecordCursor#getCompletedBytes`'s javadoc doesn't mention 0 as the special value.
In fact, it doesn't exist.

But, JDBC connectors do return 0.

Reply:
Let me send another PR for javadoc. Also, I will fix MongoDB connector too.